src/SALib/analyze/morris.py

```diff
@@ -135,7 +140,13 @@ def analyze(
     num_trajectories = int(Y.size / (number_of_groups + 1))
     trajectory_size = int(Y.size / num_trajectories)
 
-    elementary_effects, see_trajectory = _compute_elementary_effects(X, SX, Y, trajectory_size, delta)
+    delta = _compute_delta(num_levels)
+    elementary_effects = _compute_elementary_effects(X, Y, trajectory_size, delta, scaling=False)
+    
+    if SX is None:
+        scaled_elementary_effects=None
```

Comment:
This should compute `scaled_elementary_effects` with X instead of SX.  E.g. default to use X to compute the scaled elementary effects and only use SX if that is provided.

Reply:
I'm not sure that I agree on this. Let's say the use case was my case, then I would send in an X ranging between [0, 1]. And from that I would get a SEE which I think would be wrong calculated. How can one prevent an analyst to use "false" numbers?

Reply:
In your case you provide SX. Otherwise you don't need to, SX=X and you still need to calculate EE.

Reply:
Ok, so the idea is to always require SX in my type of use case?

Reply:
Yes